clusterDeployer.py

```diff
@@ -33,6 +33,177 @@ def match_to_proper_version_format(version_cluster_config: str) -> str:
     return match.group(0)
 
 
+def _resolv_conf_parse_file(rc_file: str) -> list[str]:
+    try:
+        with open(rc_file, "rb") as f:
```

Comment:
why not use `Host`

Reply:
you mean, `Host.read_file()`? Why is that better?

Reply:
Because I can pass in Host, and it can be remote.

Reply:
That is not used (and probably never will). The other hosts in the cluster have their own DNS setup already (using coredns). It's unlikely, that we would touch /etc/resolv.conf on other hosts.

It's more cumbersome and less pythonic, to implement system interactions in terms of `Host.run()` (or `Host.read_file()`.